tests/test_browser.py

```diff
@@ -3891,3 +3891,6 @@ def test_unicode_html_shell(self):
     open(self.in_dir('shell.html'), 'w').write(open(path_from_root('src', 'shell.html')).read().replace('Emscripten-Generated Code', 'Emscripten-Generated Emoji 😅'))
     subprocess.check_output([PYTHON, EMCC, os.path.join(self.get_dir(), 'main.cpp'), '--shell-file', 'shell.html', '-o', 'test.html'])
     self.run_browser('test.html', None, '/report_result?0')
+
+  def test_strptime_symmetry(self):
+    self.btest('strptime_symmetry.cpp', expected='1', args=['-s', 'NO_EXIT_RUNTIME=0'])
```

Comment:
The library changes look good to me (but I admit I don't have a deep understanding of strptime...), but when I run that testcase natively using `g++`, it fails on
```
2018-03-27T19:33:09+0000: 1522207989 == 1522179189, false
```
and the reported result is 0.

Also is there a reason this is in the `browser` test suite? If not, it could be in `other` (which is faster and we run in more places).